addressable.gemspec

```diff
@@ -24,14 +24,14 @@ Gem::Specification.new do |s|
     s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<public_suffix>, ["< 4.0", ">= 2.0.2"])
+      s.add_runtime_dependency(%q<public_suffix>, ["< 4.1", ">= 2.0.2"])
       s.add_development_dependency(%q<bundler>, ["< 3.0", ">= 1.0"])
     else
-      s.add_dependency(%q<public_suffix>, ["< 4.0", ">= 2.0.2"])
+      s.add_dependency(%q<public_suffix>, ["< 4.1", ">= 2.0.2"])
```

Comment:
These should probably all just be < 5 since 4.x should be compatible with 4.0

Reply:
Agree! Changed 👍